Ska/ftp.py

```diff
@@ -180,6 +180,8 @@ def __getattr__(self, attr):
         """
         Fall through to SFTPClient methods, and fail if not found.
         """
+        if (attr == 'ftp') or (attr == 'logger'):
```

Comment:
I borrowed this idea to just check for == to 'ftp' or 'logger' from:

http://nedbatchelder.com/blog/201010/surprising_getattr_recursion.html

I think it makes sense for our use cases.